src/gc/env/gcenv.os.h

```diff
@@ -138,7 +138,7 @@ class GCToOSInterface
     //                       on output, the number of page addresses that are returned in the array.
     // Return:
     //  true if it has succeeded, false if it has failed
-    static bool GetWriteWatch(bool resetState, void* address, size_t size, void** pageAddresses, uintptr_t* pageAddressesCount);
+    static void GetWriteWatch(bool resetState, void* address, size_t size, void** pageAddresses, uintptr_t* pageAddressesCount);
```

Comment:
All of the callers were just asserting that the return value is true, so I don't see any reason for the return value. Anyway, this is not important to my change, reverted.